.github/CONTRIBUTING.md

```diff
@@ -11,6 +11,10 @@ We welcome contributions from everyone. However, please follow the following gui
 
 Following these guidelines makes it easier for us to process your issue, and ensures we won't close your issue right-away for being misfiled.
 
+## Security vulnerabity reports
+
+If you discover a security vulnerability, we'd appreciate a non-public disclosure. The issue tracker and mailing list listed above are fully public. If you need to reach systemd developers in a non-public way, open a bug in one of the "big" distributions using systemd: Fedora (https://bugzilla.redhat.com/enter_bug.cgi?product=Fedora&component=systemd, be sure to check "Security Sensitive Bug" under "Show Advanced Fields"), or Debian/Ubuntu (using reportbug, be sure to check "This bug is a security vulnerability"). Various systemd developers are active distribution maintainers and will propagate the information about the bug to other parties.
+
```

Comment:
This is pretty much only shown in GitHub, so I think we should make better usage of link formatting. Also, Ubuntu does not use `reportbug` and it's not really that easy to use on Debian either. I'd suggest

... way, report the issue in one of the "big" distributions using systemd: [Fedora](https://bugzilla.redhat.com/enter_bug.cgi?product=Fedora&component=systemd) (be sure to check "Security Sensitive Bug" under "Show Advanced Fields"), [Ubuntu](https://launchpad.net/ubuntu/+source/systemd/+filebug) (be sure to change "This bug contains information that is" from "Public" to "Private Security"), or [Debian](mailto:security@debian.org).

WDYT?  I'm happy to force-push over your branch, but I don't want to do this without your consent.